sdk/agenta/sdk/engines/running/handlers.py

```
@@ -148,109 +147,46 @@ def _compute_similarity(embedding_1: List[float], embedding_2: List[float]) -> f
     return dot / (norm1 * norm2)
 
 
-# ========= Scheme detection =========
-
-
+# Resolvers used by webhook/match evaluators below. Substitution for prompt
+# templates lives in `agenta.sdk.utils.templating.render_template`.
 from agenta.sdk.utils.resolvers import (  # noqa: E402
```

Flake8 (7.3.0)

[error] 'agenta.sdk.utils.resolvers.detect_scheme' imported but unused

(F401)